code/modules/jobs/job_types/paramedic.dm

```diff
@@ -1,12 +1,12 @@
 /datum/job/paramedic
-	title = "Paramedic"
+	title = "Field Surgeon"
```

Comment:
Paramedic name suggestion - Leecher.

Reply:
see my response to the other review about changing jailer to gaoler